### PR TITLE
Misc cleanup

### PR DESCRIPTION
**Pull request by ekoby**
* update copyright notices
* mode dependencies into subdir

### CODE REVIEW

File: deps/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+set(ENABLE_PROGRAMS OFF CACHE BOOL "" FORCE)
+set(ENABLE_TESTING OFF CACHE BOOL "" FORCE)
```
Comment (dovholuknf) [inline deps/CMakeLists.txt]: tests are off by default?

File: deps/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+add_subdirectory(mbedtls)
+set_target_properties(mbedtls PROPERTIES
+        INTERFACE_INCLUDE_DIRECTORIES ${CMAKE_CURRENT_BINARY_DIR}/mbedtls/include)
+
+set(BUILD_TESTING OFF CACHE BOOL "" FORCE)
+set(ENABLE_PROGRAMS OFF CACHE BOOL "" FORCE)
```
Comment (dovholuknf) [inline deps/CMakeLists.txt]: ENABLE_PROGRAMS duplicate?

File: deps/CMakeLists.txt (hunk 1 above)
Comment (ekoby) [reply deps/CMakeLists.txt]: for dependencies? yes

File: deps/CMakeLists.txt (hunk 2 above)
Comment (ekoby) [reply deps/CMakeLists.txt]: this was just copied from top-level CMakeLists.txt

**Pull request merged**